.circleci/config.yml

```diff
@@ -117,6 +117,8 @@ jobs:
           cmd: go mod download
       - docker-run:
           cmd: make circleci-test build
+          options:
+            --env=GITHUB_TOKEN=$EKSCTLBOT_PAT
```

Comment:
I'm unnaturally bothered by this not having a `_TOKEN` suffix like `WEAVEWORKSBOT_TOKEN`

Reply:
😂 

Which would you prefer: `EKSCTLBOT_PERSONAL_ACCESS_TOKEN` or `EKSCTLBOT_PA_TOKEN`. I will not do `EKSCTLBOT_PAT_TOKEN` because that is even more upsetting.

Reply:
`EKSCTLBOT_TOKEN`?